samples/csharp/sharedcontent/console/speech_synthesis_server_scenario_sample.cs

```diff
@@ -39,16 +39,22 @@ public class SynthesizerPool : IDisposable
         private readonly int _initialCapacity;
         private readonly int _maximumRetainedCapacity;
 
-        public SynthesizerPool(Func<SpeechSynthesizer> synthesizerGenerator, int initialCapacity = 2, int maximumRetainedCapacity = 64)
+        public SynthesizerPool(Func<SpeechSynthesizer> synthesizerGenerator, int initialCapacity = 2, int maximumRetainedCapacity = 100)
         {
             _synthesizerGenerator = synthesizerGenerator;
             _synthesizerStack = new ConcurrentStack<SpeechSynthesizer>();
             _initialCapacity = initialCapacity;
             _maximumRetainedCapacity = maximumRetainedCapacity;
+
+            Console.WriteLine($"Create initial {initialCapacity} syntheszier and warm up");
             for (var i = 0; i < initialCapacity; i++)
             {
-                Put(_synthesizerGenerator());
+                SpeechSynthesizer item = _synthesizerGenerator();
+                item.SpeakTextAsync("1").Wait();
```

Comment:
Can you add a comment or print a message to illustrate the purpose of this speak?